test/unittests/instructions_test.cpp

```diff
@@ -136,9 +144,14 @@ TEST(instructions, compare_undefined_instructions)
 
         for (size_t i = 0; i < instr_tbl.size(); ++i)
         {
+            // TODO pending update in EVMC
+            if (r >= EVMC_CANCUN && (i == OP_RJUMP || i == OP_RJUMPI || i == OP_RJUMPV ||
+                                        i == OP_CALLF || i == OP_RETF))
+                continue;
+
             // Skip DUPN and SWAPN. They are not defined in evmc
             // TODO: Define DUPN and SWAPN in evmc
-            if (Opcode(i) == OP_DUPN || Opcode(i) == OP_SWAPN)
+            if (r >= EVMC_CANCUN && (Opcode(i) == OP_DUPN || Opcode(i) == OP_SWAPN))
```

Comment:
Same here.

Reply:
Can we just create a helper on top of this file for `evmone_only_instruction()`?

Reply:
Added.